libstemmer/modules.txt

```diff
@@ -28,6 +28,7 @@ italian         UTF_8,ISO_8859_1        italian,it,ita
 lithuanian      UTF_8                   lithuanian,lt,lit
 nepali          UTF_8                   nepali,ne,nep
 norwegian       UTF_8,ISO_8859_1        norwegian,no,nor
+polish          UTF_8                   polish,pl,pol
```

Comment:
I think this could be `UTF_8,ISO_8859_2` (if we're going to have ISO-8859-2 support in libstemmer, we ought to have it available for languages for which it covers the alphabet).